tasks/init.json

```
@@ -0,0 +1,41 @@
+{
+  "description": "Make a HTTP or HTTPS request.",
+  "parameters": {
+    "base_url": {
+      "description": "The fully qualified URL scheme to make requests to.",
+      "type": "String[1]"
+    },
+    "body": {
+      "description": "The request body.",
+      "type": "Optional[String]"
+    },
+    "cacert": {
+      "description": "An absolute path to the CA certificate.",
+      "type": "Optional[String[1]]"
+    },
+    "cert": {
+      "description": "An absolute path to the client certificate.",
+      "type": "Optional[String[1]]"
+    },
+    "headers": {
+      "description": "A map of headers to add to the payload.",
+      "type": "Optional[Hash[String, String]"
```

```suggestion
      "type": "Optional[Hash[String, String]]"
```

This causes an extremely obtuse Puppet error, for the record... "Syntax error at end of input" with no context.

---

🤦